techniques/systemSettings/systemManagement/fstabConfiguration/4.0/metadata.xml

```diff
@@ -1,6 +1,7 @@
 
 <TECHNIQUE name="Filesystem mount points">
   <DESCRIPTION>Configures the system fstab</DESCRIPTION>
+  <DEPRECATED>This technique version is not maintained anymore, it will no longer be available in the next stable version of Rudder.</DEPRECATED>
```

Comment:
```suggestion
  <DEPRECATED>This technique version is not maintained anymore, it will no longer be available in the next version of Rudder.</DEPRECATED>
```